src/ServiceControl.Transports/BrokerThroughput/IBrokerThroughputQuery.cs

```diff
@@ -20,4 +20,5 @@ public interface IBrokerThroughputQuery
     KeyDescriptionPair[] Settings { get; }
     Task<(bool Success, List<string> Errors, string Diagnostics)> TestConnection(CancellationToken cancellationToken);
     string SanitizeEndpointName(string endpointName);
+    string SanitizedEndpointNameCleanser(string endpointName);
```

Comment:
Do we need another method? Is it not possible for `SanitizeEndpointName` to include any necessary "cleansing"?

Reply:
no as that's used for something else - to store the sanitized name. We are then using the cleansing to cleanse the audit and monitoring sanitized name to that it matches what the broker has.

Reply:
yeah, i was thinking the same but i can see the reason